audio_service/lib/audio_service.dart

```diff
@@ -2190,16 +2216,10 @@ class _IsolateAudioHandler extends AudioHandler {
 }
 
 /// Base class for implementations of [AudioHandler]. It provides default
-/// implementations of all methods and provides controllers for emitting stream
-/// events:
+/// implementations of all methods and provides various [BehaviorSubject]s, that contain values
+/// and behave like streams: [playbackState], [queue] and [mediaItem].
 ///
-/// * [playbackStateSubject] is a [BehaviorSubject] that emits events to
-/// [playbackStateStream].
-/// * [queueSubject] is a [BehaviorSubject] that emits events to [queueStream].
-/// * [mediaItemSubject] is a [BehaviorSubject] that emits events to
-/// [mediaItemStream].
-/// * [customEventSubject] is a [PublishSubject] that emits events to
-/// [customEvent].
+/// The [customEventSubject] is is a [PublishSubject] that emits events to [customEvent].
```

Comment:
"is is"

Not sure why you kept this line but removed the others.

Reply:
i didn't remove them, but moved them above. i kept this one because it has different type